src/SonarScanner.MSBuild.Shim/PropertiesFileGenerator.cs

```diff
@@ -152,6 +146,120 @@ public bool TryWriteProperties(PropertiesWriter writer, out IEnumerable<ProjectD
             return true;
         }
 
+        internal /* for testing */ static ProjectData GetSingleClosestProjectOrDefault(FileInfo fileInfo, IEnumerable<ProjectData> projects)
```

Comment:
did you modify anything in lines 149 - 261 or just copy pasted from down below?

(to know whether to review them)

Reply:
I think I've touched every method

Reply:
(..I should have create a separte "move" commit 😒 )